addresses.json

```diff
@@ -1,5 +1,5 @@
 {
     "localhost": {
-        "NFTOpt": "Your NFTOpt contract address deployed on localhost"
+        "NFTOpt": "0x5FbDB2315678afecb367f032d93F642f64180aa3"
```

Comment:
perhaps remove this file from commit, and also add it to git ignore, somehow? need to discuss how to manage this